contracts/script/test/SendCrossChainMessageExample.s.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.25;
+
+import { Script, console2 } from "forge-std/Script.sol";
+import { CrossChainExample } from "../deploy/DeployCrossChainExample.s.sol";
+import { T1Constants } from "../../src/libraries/constants/T1Constants.sol";
+
+contract SendCrossChainMessageExample is Script {
```

Comment:
Does it make sense to add this new test to the README Test's section as well? https://github.com/t1protocol/t1/tree/canary/contracts/script#test

Reply:
This is not a test though. I think we should keep it separate. Otherwise we would need to add all other scripts in this `scripts/test` folder, which are also not really tests.